lib/lte_link_control/lte_lc.c

```diff
@@ -799,6 +799,16 @@ int lte_lc_power_off(void)
 	return 0;
 }
 
+int lte_lc_deinit(void)
+{
+	if (is_initialized) {
+		is_initialized = false;
+		return lte_lc_power_off();
```

Comment:
Could there be situations where a user would not want to power off the modem at the same time?
Just a bit unsure if we implicitly want to power off at the same time.

Reply:
I don't see any situation where that make sense. Of course you can technically still use the modem after you call lte_lc_deinit();. But if lte_lc is your link control API it doesn't make sense that the link should be kept up after you call deinit.

Reply:
Agreed